packages/flutter/test/material/ink_paint_test.dart

```diff
@@ -9,14 +9,19 @@ import 'package:flutter_test/flutter_test.dart';
 import '../rendering/mock_canvas.dart';
 
 void main() {
-  testWidgets('The Ink widget renders a Container by default', (WidgetTester tester) async {
+  testWidgets('The Ink widget renders a SizedBox by default', (WidgetTester tester) async {
     await tester.pumpWidget(
       Material(
         child: Ink(),
       ),
     );
-    expect(tester.getSize(find.byType(Container)).height, 600.0);
-    expect(tester.getSize(find.byType(Container)).width, 800.0);
+    Finder sizedBox = find.byWidgetPredicate(
+      (Widget widget) => widget is SizedBox
+          && widget.height == null
+          && widget.width == null
```

Comment:
Did you have to add the `height == null && width == null` because it was finding multiple SizedBoxes otherwise?  If so, instead you could consider using `find.descendant` of Ink.  Here and below.

Reply:
Ah, yes indeed - much more elegant. Thanks!

Reply:
Maybe I got excited too quickly - in the second case (with provided size), Ink render this:

```
        └Ink(state: _InkState#07027)
         └SizedBox(width: 200.0, height: 150.0, renderObject: RenderConstrainedBox#2c84a relayoutBoundary=up1)
          └Padding-[GlobalKey#8689d](padding: EdgeInsets.zero, renderObject: RenderPadding#3364f)
           └Builder
            └SizedBox(renderObject: RenderConstrainedBox#a918b)
```

Let's pick the first one.